tldw_Server_API/app/core/DB_Management/Evaluations_DB.py

```
@@ -256,12 +256,104 @@ def get_connection(self):
 
         if self.backend is None:
             raise RuntimeError("Evaluations backend is not configured")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)